frr_exporter.go

```diff
@@ -20,9 +21,9 @@ import (
 )
 
 var (
-	listenAddress = kingpin.Flag("web.listen-address", "Address on which to expose metrics and web interface.").Default(":9342").String()
 	telemetryPath = kingpin.Flag("web.telemetry-path", "Path under which to expose metrics.").Default("/metrics").String()
 	configFile    = kingpin.Flag("web.config", "[EXPERIMENTAL] Path to config yaml file that can enable TLS or authentication.").Default("").String()
```

Comment:
The `configFile` flag is now redundant, since the flag is added by `kingpinflag.AddFlags()`.

Reply:
Just cleaned that up.